src/_gettsim/synthetic.py

```diff
@@ -172,7 +172,12 @@ def create_constant_across_households_variables(df, n_adults, n_children, policy
         datetime.date(policy_year, 1, 1),
         RESOURCE_DIR / "synthetic_data" / "bedarfsgemeinschaften",
     )
-    hh_typ_string = create_hh_typ_string(n_adults, n_children)
+
+    # Use data for 2 children if there are more than 2 children in the household.
+    hh_typ_string = (
```

Comment:
Is there a reason why you don't use the `create_hh_typ_string` function here?

Reply:
Good point! I adjusted it in the last commit.